linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/oval/shared.xml

```diff
@@ -0,0 +1,44 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("All password change date is in the past.") }}}
+    <criteria>
+      <criterion comment="passward last change was in the past" test_ref="test_accounts_password_last_change_time_secs"/>
```

Comment:
Could you adjust the OVAL so it fit in 99 columns, please?

Reply:
:+1: changed in [135d663](https://github.com/ComplianceAsCode/content/pull/10031/commits/135d663c3dac8a18fb51088c0b758f095081c5fd)